sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,252 @@
+
+The following eligibility criteria will be used to identify the sBTC Signers:
+
+- Does the proposed sBTC signer have a demonstrable operating history which shows their experience and reliability in running blockchain services?
+- Has the proposed sBTC signer participated in running a Stacks signer instance on Stacks 2.5 testnet or mainnet, and can they provide metrics showing this (ex: amount of stacks stacked over past several cycles)?
+  * Note: The sBTC signer is a Stacks event observer, meaning that the experience of running a Stacks node signer directly translates to running an sBTC signer.
+- Does the proposed sBTC signer agree to use reasonable efforts to maintain >99% uptime on the sBTC Signer? 
+  * Note: This metric may be self-affirmed if independent verification is not possible, or confirmed by on-chain voting/stacking activity.
+- Does the proposed sBTC signer commit to a direct communication channel to be set up with the sBTC core engineers in order to respond to urgent updates within 24 hours?
+- Has the signer made contributions to Bitcoin or the Stacks network over the past year that demonstrate their commitment to the growth and success of the network? Examples include, but are not limited to: publishing independent research, marketing, co-authoring a SIP, submitting a Stacks pull request/issue, providing feedback on Stacks core development, or contributing to a Stacks Working Group.
+
+The criteria described above will be used to identify sBTC Signers that are able to meet some or all of the responsibilities described in the previous section.
+
+### Selection Process
+The sBTC Signer Set will be finalized from the list of eligible Signers, based on the above criteria. The [sBTC Working Group](https://github.com/orgs/stacks-network/discussions/508) will conduct the vetting process and the results will be published as a discussion in the [sBTC Github repository](https://github.com/stacks-network/sbtc/discussions/624). 
+
+The selection process is as follows:
+1. **Nomination Phase**: Open a call for nominations within the community.
+2. **Evaluation & Community Feedback**: The proposed signer set will be published to provide transparency.
+3. **SIP Vote**: The community will vote on the sBTC signer criteria.
+4. **Final Selection**:  If SIP-028 is ratified, then the proposed signer set voted upon in step 3 shall be the initial signing set for sBTC. If the signer set changes during the vote, such as by the withdrawal of one or more candidates, then the vote will be restarted in a subsequent reward cycle (to be determined if this comes to pass).
+
+   
+### Updating The sBTC Signer Set
+In the event that the sBTC Signer Set needs to be updated (for example, if a signer is no longer available to complete their responsibilities) sBTC Signers can perform a threshold vote to agree on the updated set. This process will also be performed if a signer needs to rotate their cryptographic keys.
```

Comment:
is the threshold same as the threshold for approving pegging transactions? if yes best to state?
As it does not necessarily have to be the same.